src/components/Books/Components.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+import {faStar, faStarHalf} from '@fortawesome/free-solid-svg-icons';
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
+
+const UserLabel = props => (
+  <div className="user-label">
+    <img src={props.avatarURL} />
+    <div>
+      <div>{props.userName}</div>
+      <div>{props.bio}</div>
+    </div>
+  </div>
+);
+
+const RatingStars = props => {
+  // Round to nearest half
+  const rating = Math.round(props.rating * 2) / 2;
+  let output = [];
+  // Append all the filled whole stars
+  for (let i = rating; i > 0; i--)
+    // If there is a half a star, append it
+    if (i === 0.5) {
+      output.push(<FontAwesomeIcon icon={faStarHalf} color="#FFAC2D" />);
```

Comment:
Insteadf of using the color attribute here, I would style the SVG in an SCSS stylesheet.